client/my-sites/site-indicator/index.jsx

```diff
@@ -55,20 +52,6 @@ class SiteIndicator extends Component {
 		return this.props.siteIsConnected === false;
 	}
 
-	hasWarning() {
```

Comment:
An idea to consider: what if we preserved the "warning" functionality, keeping it unused? Sooner or later, we might want to start showing some site warning again.

Reply:
Good call, but I'd say YAGNI 😉 If we need it, we can always re-add it; and I usually prefer removing dead code rather than keeping it.